packages/wrangler/src/inspect.ts

```diff
@@ -667,6 +552,125 @@ export default function useInspector(props: InspectorProps) {
 	]);
 }
 
+export async function logUserWorkerException(
```

Comment:
No changes to this code other than extracting it into its own function

Reply:
No longer relevant since we rebased and the code (2 lines) has been moved into ProxyController